src/components/v1/V1Create/index.tsx

```diff
@@ -690,16 +708,14 @@ export default function V1Create() {
           {...drawerStyle}
           onClose={() => {
             viewedCurrentStep()
+            resetIncentiveForm()
             setIncentivesFormModalVisible(false)
+            setIncentivesFormState(incentivesFormState + 1)
           }}
         >
           <IncentivesForm
```

Comment:
Nice @johnnyd-eth! Seems to work great. What do you think about using a pattern like this instead of `formState`? This causes the entire form component to re-render whenever the modal is closed. Still kinda awk, but feels a bit more lightweight and would save us having to pass `formState` down through so many components

```suggestion
{incentivesFormModalVisible &&
          <IncentivesForm
```

Reply:
Ahh yeah that's way better lol I over-thunk it. Cheers Peri!